content/influxdb/v2.3/reference/release-notes/influxdb.md

```diff
@@ -0,0 +1,1816 @@
+---
+title: InfluxDB v2.3 release notes
+description: Important changes and what's new in each version of InfluxDB.
+menu:
+  influxdb_2_3_ref:
+    name: InfluxDB
+    parent: Release notes
+weight: 101
+---
+
+## v2.3 [2022-6-17]
+
+This release includes the following [maintenance](#maintenance), [features](#features) and [bug fixes](#bug-fixes).
+
+### Maintenance
+
+- Upgrade from Flux 0.161.0 to [Flux 0.171.0](/flux/v0.x/release-notes/#v01710-2022-06-14).
+- Upgrade to [Go 1.18.3](https://go.dev/doc/go1.18).
+
+### Features
+
+- Add the [`--instance-id`](/influxdb/v2.3/reference/config-options/?t=JSON#instance-id) flag to `influxd` runtime to add the `_instance_id` tag to remote replications metrics. Helps to identify where the metrics came from, and ensures that two instances don't overwrite each others' data.
+- Add signifier to replication `user-agent` to signify when replication write requests are made. Helps to identify metrics written via replications.
+
+#### Flux advancement highlights
+
+##### Performance improvements
+
+- Vectorize arithmetic operators in `map()`.
+- Vectorize logical operations in `map()`.
+- Enable `movingAverage()` and `cumulativeSum()` optimizations by default.
+
+##### Other highlights
+
+- Add `preview()` to experimental package for limiting return rows and tables (as opposed to just rows with `limit()`).
+- Add `date.scale()` to let users dynamically scale durations in dates.
+- Add [OpenTracing](https://opentracing.io/docs/overview/spans/) spans to Flux transformations. Lets you monitor Flux scripts more precisely.
+- Add `trace` option to Flux CLI.
+- Move `addDuration()` and `subDuration()` out of experimental to date package.
```

Comment:
@sanderson @sunbryely-influxdata looks like we need to move these 2 functions in docs

Reply:
@kelseiv just checked on this and we have already moved the `addDuration` `and subDuration` into the dates package. they were renamed from `addDuration` to `add` upon promotion to the date package

Reply:
Great news, thanks @sunbryely-influxdata --I'll clarify in the release notes